checker/checker.go

```diff
@@ -64,13 +64,15 @@ func Check(parsedExpr *exprpb.ParsedExpr, source common.Source, env *Env) (*ast.
 	for id, t := range c.types {
 		m[id] = substitute(c.mappings, t, true)
 	}
+	e, err := ast.ProtoToExpr(parsedExpr.GetExpr())
+	if err != nil {
 
-	return &ast.CheckedAST{
-		Expr:         parsedExpr.GetExpr(),
-		SourceInfo:   parsedExpr.GetSourceInfo(),
-		TypeMap:      m,
-		ReferenceMap: c.references,
-	}, errs
+	}
+	info, err := ast.ProtoToSourceInfo(parsedExpr.GetSourceInfo())
+	if err != nil {
+
```

Comment:
?

Reply:
Missed this when I was trying to break up the larger PR into smaller ones. Thank you.